docs/manual-CN/2.query-language/5.appendix/gremlin-ngql.md

```diff
@@ -0,0 +1,243 @@
+  ==>v[1]
+  gremlin> v2 = g.addV("character").property(T.id, 2)property('name','jupiter').property('age', 5000).property('type', 'god');
+  ==>v[2]
+
+  # 插入边
+  nebula> INSERT EDGE father() VALUES hash("jupiter")->hash("saturn"):();
+  gremlin> g.addE("father").from(v1).to(v2).property(T.id, 3)
+  ==>e[3][1-father->2]
+  ```
+
+- 删除数据
+  
+  ```bash
+  nebula> DELETE VERTEX hash("prometheus")
+  gremlin> g.V(v3).drop()
+  ```
+
+- 更新数据
+
+```bash
+nebula> UPDATE VERTEX hash("jesus") SET character.type = 'titan';
+gremlin> g.V(v7).property('age', 6000)
+```
+
+- 查看数据
+  
+  ```bash
+  nebula> FETCH PROP ON character hash("saturn");
+  ===================================================
+  | character.name | character.age | character.type |
+  ===================================================
+  | saturn         | 10000         | titan          |
+  ---------------------------------------------------
+
+  gremlin> g.V(1).valueMap()
+  ==>[name:[saturn],type:[titan],age:[10000]]
+  ```
+
+- 查询 hercules 的祖父
+
+    ```bash
+    nebula> GO 2 STEPS FROM hash("hercules") OVER father YIELD  $$.character.name
+    =====================
+    | $$.character.name |
+    =====================
+    | saturn            |
+    ---------------------
+
+    gremlin> g.V().hasLabel('character').has('name','hercules').out ('father').out('father').value('name')
+    ==>saturn
+    ```
+
+- 查询 hercules 的父亲
+
+    ```bash
+    nebula> GO FROM hash("hercules") OVER father YIELD $$.character.name
+    =====================
+    | $$.character.name |
+    =====================
+    | jupiter           |
+    ---------------------
+
+    gremlin> g.V().hasLabel('character').has('name','hercules').out ('father').value('name')
+    ==>jupiter
+    ```
+
+- 查询年龄大于 100 的人物
+
+    ```bash
+    nebula> XXX # not supported yet
+    gremlin> g.V().hasLabel('character').has('age',gt(100))
+    ```
+
+- 查询和 pluto 一起居住的人物
+
+    ```bash
+    nebula> GO FROM hash("pluto") OVER lives YIELD lives._dst AS place | \
+                          GO FROM $-.place OVER lives REVERSELY YIELD \
+                          $$.character.name AS cohabitants
+    ===============
+    | cohabitants |
+    ===============
+    | pluto       |
+    ---------------
+    | cerberus    |
+    ---------------
+
+    gremlin> g.V(pluto).out('lives').in('lives').values('name')
+    ==>pluto
+    ==>cerberus
+    ```
+
+- 从一起居住的人物中排除 pluto 本人
+
+    ```bash
+    nebula>  GO FROM hash("pluto") OVER lives YIELD lives._dst AS place | \
+                        GO FROM $-.place OVER lives REVERSELY WHERE \
+                        $$.character.name != "pluto" YIELD \
+                        $$.character.name AS cohabitants
+    ===============
+    | cohabitants |
+    ===============
+    | cerberus    |
+    ---------------
+
+    gremlin> g.V(pluto).out('lives').in('lives').where(is(neq(pluto)))  .values('name')
```

Comment:
extra space

Reply:
![image](https://user-images.githubusercontent.com/42762957/71456417-4dcc3500-27d4-11ea-85ea-cc373268d796.png)
the preview is what I expected